src/components/Navigator/NavigatorDataProvider.vue

```diff
@@ -99,7 +99,7 @@ export default {
         );
         this.navigationIndex = Object.freeze(interfaceLanguages);
         this.navigationReferences = Object.freeze(references);
-        AppStore.setIncludedArchiveIdentifiers(includedArchiveIdentifiers);
+        IndexStore.setIncludedArchiveIdentifiers(includedArchiveIdentifiers);
```

Comment:
Is this line still needed since this state is now being set in the view component? I know this component will likely be removed in the future, but I think the code using these identifiers are not tied to this provider like the navigator stuff is.